tasks/ldap.yml

```diff
@@ -0,0 +1,20 @@
+---
+- name: Grafana | LDAP | Ensure config directory
+  file:
+    dest: "{{ grafana_ldap_host_config_path }}"
+    state: directory
+    mode: 0755
+    owner: root
+    group: root
+
+- name: Grafana | LDAP | Ensure LDAP config
+  template:
+    src: config/ldap.toml.j2
+    dest: "{{ grafana_ldap_host_config_path }}/ldap.toml"
+    mode: 0644
+    owner: root
+    group: root
+
+- name: Grafana | LDAP | Set volumes with ldap config
```

Comment:
You do not actually need this. Just pass host config path to container one always.